ipam/allocator.go

```diff
@@ -397,20 +402,34 @@ func (a *Allocator) getPredefinedPool(as string, ipV6 bool) (*net.IPNet, error)
 		return nil, err
 	}
 
-	for _, nw := range a.getPredefineds(as) {
+	pdl := a.getPredefineds(as)
+
+	aSpace.Lock()
+	defer aSpace.Unlock()
+
+	c := a.asIndices[as]
```

Comment:
maybe silly but can we give a more meaningful name to this variable?
something like addressSpaceIndex or nextNetworkIdx?

Reply:
Are you referring to `c` or to `asIndices`?  If the former, I'd chafe a bit at such a long name for an index variable.  If the latter, I'd have expected the comment to be at its declaration.  Hence the confusion.

Reply:
I was referring to `c`